lib/commands/screenshot.js

```diff
@@ -0,0 +1,11 @@
+let commands = {};
```

Comment:
What if, rather than stopping proxying and doing everything manually (because every command that uses WDA is going to have to be duplicated in this scheme), we make use of the `appium-safari-driver` NO_PROXY list and whatnot. That is, when we are in a Safari/Webview context, calls to `proxyActive`, `canProxy`, and `getProxyAvoidList` defer to SafariDriver?

Then we would have something like 

```js
class XCUITestDriver extends BaseDriver {
  // ...

  getProxyAvoidList () {
    if (this.inWebview()) {
      return this._safariDriver.getProxyAvoidList();
    } else {
      // do what we now do
    }

  // ...
}
```

This is a bit more work in the short term, but to my mind it keeps the logic more close to the logic of a normal driver. The separation between the delegated driver and the normal driver is less.

But I'm perfectly happy to entertain dissenting opinions! :)